File: bubble sorting/1/1.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>



int main(){

    char choice [12];
    int x;
    int y;

    printf("Welcome to simple calculator\n");
    printf("\n");

    printf("Do you want to add subtract or divide? :\n");

    scanf("%s", choice);


    // switch (choice){ 
    //     case 'add' :
    //     printf("enter your first number: \n");
    //     scanf("%d", &x);
    //     printf("enter your second number: \n");
    //     scanf("%d", &y);
    //     printf("the sum is %d", x + y);

        
    // }
    // so i found out that you cant compare strings in using the switch statement its only used for ints and char
    //and also you dont need to add & to an array just do scanf("%s", <array name no '&'>)



    if (strcmp(choice, "add")== 0)
    {
        printf("you want to add ?");
    }
    else if(strcmp(choice, "subtract")== 0)
    {
        printf("you want to subtract");
    }
    else if(strcmp(choice, "divide") == 0)
    {
        printf("you want to divide");
    }else{
        printf("Invalid operation selected");
    }


    



    return 0;
}

File: bubble sorting/1/1.py

print("Welcome to simple calculator\n")
print("\n")
choice = input("do you want to add, subract, multiply or divide? :")


try:
    if choice.lower() == "add":
        while True:
            
                x = input("enter you first number:")
                y = input("enter your second number:")

                if x.isdigit() and y.isdigit():
                     print(int(x)+int(y))
                     break
                
                else:
                     print(f"{x} or {y} is not a valid integer")
    elif choice.lower() == "subtract":
         while True:
            
                x = input("enter you first number:")
                y = input("enter your second number:")

                if x.isdigit() and y.isdigit():
                     print(int(x)-int(y))
                     break
                
                else:
                     print(f"{x} is not a valid integer")
    elif choice.lower() == "multiply":
         while True:
            
                x = input("enter you first number:")
                y = input("enter your second number:")

                if x.isdigit()  and y.isdigit():
                     print(int(x)*(y))
                     break
                
                else:
                     print(f"{x} is not a valid integer")
    elif choice.lower() == "divide":
         while True:
              
              x = input("enter your first number: ")
              y = input("enter you second number: ")

              if x.isdigit() and y.isdigit():
                   print(int(x)/int(y))
                   break
              else:
                   print("not a valid integer")
         
                


            


except Exception as e:
    print (e)

